src/codegen/function_body_chunk.rs

```diff
@@ -182,9 +200,13 @@ impl Builder {
                      type_: None,
                  });
         body.extend(output_vars);
+        let ret_name = match trampoline.use_ret {
+            true => "retval",
+            false => "_",
+        };
```

Comment:
Please just "ret" and use if

Reply:
done, though I'm not sure using if looks much better?